docs/data/material/design-resources/material-ui-sync/material-ui-sync.md

```diff
@@ -56,13 +56,12 @@ After you've added your custom tokens, click on **Regenerate theme** to include
 
 ## Customizing components
 
-The Sync plugin can also generate theme styles for customized components, enabling you to completely change their look and feel and create your custom design system from within Figma.
-
-:::info
-This feature is currently limited to the Button, Switch, and Typography components.
-Support for more components is coming soon.
+:::warning
+This feature is experimental and limited to the Button, Switch, and Typography components.
```

Comment:
Maybe we could make it even clearer that only three components are currently supported:

```suggestion
This feature is experimental and limited to the **Button**, **Switch**, and **Typography** components.
```